sql/postgres/convert.go

```diff
@@ -181,6 +181,86 @@ func ParseType(typ string) (schema.Type, error) {
 	return t, nil
 }
 
+func columnType(c *columnDesc) (schema.Type, error) {
```

Comment:
Move from a `inspect.go` to `convert.go` close to the rest of the parsing/formatting functions.